roles/setup/tasks/main.yml

```diff
@@ -144,37 +144,60 @@
     - setup_reports
     - oc_project.stdout is defined and oc_project != ''
 
+- name: Check for hccm-openshift-persistentvolumeclaim Report
+  command: '{{ OCP_CLI }} get Report {{ setup_storage_report_name }}'
+  register: oc_get_storage_report
+  changed_when: false
+  ignore_errors: true
+  when:
+    - setup_reports
+    - oc_project.stdout is defined and oc_project != ''
+
+- name: Check for hccm-openshift-persistentvolumeclaim ReportGenerationQuery
+  command: '{{ OCP_CLI }} get ReportGenerationQuery {{ setup_storage_report_name }}'
+  register: oc_get_storage_report_generation_query
+  changed_when: false
+  ignore_errors: true
+  when:
+    - setup_reports
+    - oc_project.stdout is defined and oc_project != ''
+
+- name: Define temp report file data location
+  set_fact:
+    ocp_temp_file_dir: '/tmp/{{ ocp_cluster_id }}'
```

Comment:
We have been putting everything just in `/tmp` but I ran into an issue trying to setup 2 clusters on the same machine, but using different users. One user owned some of the files in /tmp, and the setup role failed because the second user could not overwrite. I figured creating a subdirectory for each cluster would avoid this